src/app/features/ahbs/views/ahb-page/ahb-page.component.spec.ts

```diff
@@ -24,7 +24,7 @@ describe('AhbPageComponent', () => {
       pruefi: '123',
     });
     const html = ngMocks.formatHtml(fixture);
-    expect(html).toContain('<app-header>');
+    expect(html).toContain('<app-header');
```

Comment:
What will happen is the string is not contained in the html?

Reply:
This unit test will fail. This is just a simple example test and could be expanded with more detailed tests